articles/app-service/azure-web-sites-web-hosting-plans-in-depth-overview.md

```diff
@@ -198,4 +198,4 @@ see: [Azure App Service and Its Impact on Existing Azure Services](http://go.mic
 [assign]: ./media/azure-web-sites-web-hosting-plans-in-depth-overview/assing-appserviceplan.png
 [change]: ./media/azure-web-sites-web-hosting-plans-in-depth-overview/change-appserviceplan.png
 [createASP]: ./media/azure-web-sites-web-hosting-plans-in-depth-overview/create-appserviceplan.png
-[createWebApp]: ./media/azure-web-sites-web-hosting-plans-in-depth-overview/create-web-app.png
\ No newline at end of file
+[createWebApp]: ./media/azure-web-sites-web-hosting-plans-in-depth-overview/create-web-app.png
```

Comment:
Not sure what changed in this line (201) can you update the PR to not have this change?